src/contracts/GPv2AccessControl.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: LGPL-3.0-or-newer
+pragma solidity ^0.6.12;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/EnumerableSet.sol";
+import "./interfaces/GPv2Authentication.sol";
+
+/// @title Gnosis Protocol v2 Access Control Contract
+/// @author Gnosis Developers
+contract GPv2AccessControl is Ownable, GPv2Authentication {
+    using EnumerableSet for EnumerableSet.AddressSet;
+
+    EnumerableSet.AddressSet private solvers;
+
+    function addSolver(address solver) public onlyOwner {
+        solvers.add(solver);
+    }
+
+    function removeSolver(address solver) public onlyOwner {
+        solvers.remove(solver);
+    }
+
+    function isSolver(address prospectiveSolver)
+        public
```

Comment:
```suggestion
        external
```

There is no reason for these to be public and not external as they aren't ever called internally.